scalyr_agent/tests/configuration_docker_test.py

```diff
@@ -92,13 +95,13 @@ def test_environment_aware_module_params(self, mock_docker):
                 if custom_name == STANDARD_PREFIX
                 else custom_name.upper()
             )
-            envar_value = str(value[1])
+            param_value = value[1]
             if value[2] == ArrayOfStrings:
                 # Array of strings should be entered into environment in the user-preferred format
                 # which is without square brackets and quotes around each element
-                envar_value = envar_value[1:-1]  # strip square brackets
-                envar_value = envar_value.replace("'", "")
+                envar_value = "[{}]".format(", ".join(param_value))
```

Comment:
When we get a string from the `list` with `unicode` elements, there is an additional `u` character at the beginning of each element. 
For example for  `str(["s1", "s2", "s3"])` becomes `'[u\'s1\', u\'s2\', u\'s3\']'` so we need to strip `u` too because config parser can not handle this.  I thought that the join will do the same job without stripping single quotes and `u`.  I have removed square brackets. I really don't know why I put them here... thanks.